arduino-ide-extension/src/browser/contributions/settings.ts

```diff
@@ -56,5 +66,11 @@ export namespace Settings {
             label: 'Open Preferences...',
             category: 'Arduino'
         }
+
+        export const CLOSE: Command = {
+            id: 'arduino-settings-close',
+            label: 'Close Preferences...',
+            category: 'Arduino'
+        }
```

Comment:
I don't think this approach is really necessary as the Dialog provides a Promise than, when resolved (usually with `close()` closes the UI.

Reply:
I agree. We do not need a command for it. Especially, a command that is available from the _Commnad Palette_. Note: if you add a `label` to the command, users can execute it from the _Commad Palette_.

Reply:
thanks for the tip about the `label`!